packages/components/src/FocusManager/FocusManager.test.js

```diff
@@ -0,0 +1,30 @@
+import FocusManager from './FocusManager.component';
+
+describe('FocusManager', () => {
```

Comment:
Instead of calling directly the class methods, can you set more real cases ?
- create test component 
- wrap it with FocusManager
- simulate a blur in the test component : check that the onFocusOut is called
- simulate a blur and focus in the test component : check that the onFocusOut is not called